src/ptvsd/adapter/ide.py

```diff
@@ -208,6 +208,8 @@ def launch_request(self, request):
             args = ["-c"] + request(
                 "code", json.array(unicode, vectorize=True, size=(1,))
             )
+        else:
+            args = []
```

Comment:
Should we fail here? if it is not a file/dir, module, or string.

Reply:
Proper handling of args, including validation, is done in launcher - this bit here is solely to generate the (dummy) args to launcher to show up in the terminal. So the part here needs to only do enough to get to the launcher - I'd rather not duplicate the validation logic.